dataland-frontend/tests/e2e/utils/GeneralUtils.ts

```diff
@@ -6,19 +6,6 @@ import { admin_name, admin_pw, uploader_name, uploader_pw } from "./Cypress";
 import { generateDummyCompanyInformation, uploadCompanyViaApi } from "./CompanyUpload";
 import { submitFilledInEuTaxonomyForm } from "./EuTaxonomyFinancialsUpload";
 
-/**
- * This function clicks "Allow all" on the Cookie Banner by Cookiebot.com to accept all optional cookies
- * Otherwise, dataland.com lacks functionality.
- */
-export function clickAllowAllOnCookieBanner(): void {
```

Comment:
Is this no longer necessary? Do we test the cookie banner in cypress?

Reply:
no the cookie banner is blocked by the githubrunner in the CI, therefore this won't succeed

Reply:
CookieBanner is only tested locally

Reply:
What do you mean by this will not succeed? Currentyl the CI is green.